lib/cretonne/src/ir/immediates.rs

```diff
@@ -558,6 +564,19 @@ impl FromStr for Ieee32 {
     }
 }
 
+impl Hash for Ieee32 {
```

Comment:
Alternatively, change the representation to `struct Ieee32(u32)` and derive these traits. I don't think we ever need that `f32` representation for anything.